components/BiospecimenTable.tsx

```diff
@@ -21,6 +21,7 @@ export const BiospecimenTable: React.FunctionComponent<IBiospecimenTableProps> =
         {
             name: 'Atlas ID',
             selector: 'atlasid',
+            format: (sample: Entity) => sample.atlasid.toUpperCase(),
```

Comment:
Should we fix this when processing the data instead?

Reply:
good point - i can prolly tweak it in the python script

Reply:
+1 - it's been confusing for me to be dealing with casing in the frontend

Reply:
fixed it here: https://github.com/ncihtan/htan-portal/pull/118

Reply:
Should we just close this PR without merging?